bitnami/postgresql-ha/values.yaml

```diff
@@ -154,9 +129,10 @@ postgresql:
   ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   ##
   updateStrategyType: RollingUpdate
```

Comment:
I think it should be like:

https://github.com/bitnami/charts/blob/5af36c98d53a5ad3911e0f4dd7fa5c63bb2b37a9/bitnami/apache/values.yaml#L357-L367

Reply:
My bad, this param should have been replaced by the one which you commented on above.

Reply:
👍